src/20_Components/amp-web-push.html

```diff
@@ -0,0 +1,69 @@
+<!---{
+  "disablePlayground": true
```

Comment:
I'm not happy with disabling the playground as it's the only way to view the whole source code on the page. I'd prefer leaving it in. I can look into why it's not working.